Config.uk

```diff
@@ -236,29 +236,78 @@ config LWIP_DNS_TABLE_SIZE
 	default 32
 endif
 
-config LWIP_SOCKET
-	bool "Socket API"
-	select LIBVFSCORE
+menu "Socket"
 	depends on LWIP_THREADS && (LWIP_UDP || LWIP_TCP)
-	default y
 
-if LWIP_SOCKET
-	config LWIP_SOCKET_SELECT_GENERIC_FDS
-		bool "Use select() with any file descriptor type"
-		depends on HAVE_LIBC
+	config LWIP_SOCKET
+		bool "Socket API"
+		select LIBVFSCORE
 		default y
+
+	if LWIP_SOCKET
+		config LWIP_SOCKET_SELECT_GENERIC_FDS
+			bool "Use select() with any file descriptor type"
+			depends on HAVE_LIBC
+			default y
+			help
+				lwip's select() implementation supports only sockets. This
+				configuration option makes it possible to use other file descriptor
+				types as well, even though they are not supported by lwip.
+
+		config LWIP_SOCKET_PPOLL
+			bool "Enable ppoll()"
+			depends on HAVE_LIBC
+			default y
+			help
+				Enable ppoll() implementation.
+	endif
+
+	config LWIP_UDP_RECVMBOX_FACTOR
+		int "Factor for UDP recvmbox"
+		range 1 5
+		default 2
 		help
-			lwip's select() implementation supports only sockets. This
-			configuration option makes it possible to use other file descriptor
-			types as well, even though they are not supported by lwip.
+			How many times should the UDP recvmbox be bigger than the LWIP stackthread mbox size.
+			Default 2, maximum 5.
 
-	config LWIP_SOCKET_PPOLL
-		bool "Enable ppoll()"
-		depends on HAVE_LIBC
-		default y
+	config LWIP_TCP_RECVMBOX_FACTOR
+		int "Factor for TCP recvmbox"
+		range 1 5
+		default 2
 		help
-			Enable ppoll() implementation.
-endif
+			How many times should the TCP recvmbox be bigger than the LWIP stackthread mbox size.
+			Default 2, maximum 5.
+endmenu
+
+choice
+    prompt "LWIP stackthread mbox size"
+    depends on LWIP_THREADS && (LWIP_UDP || LWIP_TCP)
+	default SIZE_256
```

Comment:
Please namespace these options, too, e.g.,: `LWIP_STACKTHREAD_MBOX_SIZE_256` etc.
Can you also add a helping text that says shortly descirbes what this mailbox is used for (input box for received traffic from netifs ready for getting processed by the tcpip thread (lwip stack thread))